File: STL/Arrays/Easy/3.check_if_sorted.js
const isSorted = (arr) => {
  for (let i = 1; i < arr.length; i++) {
    if (arr[i] < arr[i - 1]) return false;
  }
  return true;
};

let sorted = [1, 2, 3, 4, 5];
let unsorted = [1, 2, 4, 3, 5];

console.log(`The array ${sorted} is sorted ${isSorted(sorted)}`);
console.log(`The array ${unsorted} is sorted ${isSorted(unsorted)}`);

File: STL/Arrays/Easy/3.Check_if_Sorted.cpp
#include<iostream>
using namespace std;

int sorted(int arr[],int size){
    for(int i=1;i<size;i++){
        if(arr[i] < arr[i-1])
         return 0;
    }
    return 1;
}
int main(){
    int arr[]={1,2,3,4,5};
    int size=sizeof(arr)/sizeof(arr[0]);
    string val=sorted(arr,size)?"True":"False";
    cout<<"The Array is Sorted "<<val;
    return 0;
}